aiosmtpd/docs/intro.rst

```diff
@@ -9,7 +9,11 @@ Simple Mail Transfer Protocol (SMTP) and
 Local Mail Transfer Protocol (LMTP).  It is derived from
 `Python 3.5's smtpd.py <https://hg.python.org/cpython/file/3.5/Lib/smtpd.py>`__
 standard library module, and provides both a command line interface and an API
-for use in testing applications that send email.
+for use in testing applications that send email. Since the `smtpd module was
+removed in Python 3.12
+<https://docs.python.org/3.12/whatsnew/3.12.html#locale>`, following `PEP 594
+<https://peps.python.org/pep-0594/#smtpd>`, aiosmtpd is the recommended
```

Comment:
External references need trailing underscores in RST. The current syntax uses an anonymous role. Also, for PEPs, there's a dedicated role `:pep:`:
```suggestion
<https://docs.python.org/3.12/whatsnew/3.12.html#locale>`__, following :pep:`594#smtpd`, aiosmtpd is the recommended
```